WalletWasabi/WabiSabi/Backend/Rounds/Round.cs

```diff
@@ -103,6 +104,7 @@ public void SetPhase(Phase phase)
 		else if (phase == Phase.TransactionSigning)
 		{
 			TransactionSigningTimeFrame = TransactionSigningTimeFrame.StartNow();
+			TransactionSigningDelayTimeFrame = TimeFrame.Create(Assert<SigningState>().SigningDelay).StartNow();
```

Comment:
I would prefer much more to have the signing delay calculation here and remove the `SigningTimeProvider`.

```suggestion
			TransactionSigningDelayTimeFrame = TimeFrame.Create(CalculateSigningDelay()).StartNow();
```